lib/interpreters/periodic_visits.rb

```diff
@@ -371,36 +373,63 @@ def associate_skills(new_vehicle, vehicle_day_index)
       end
     end
 
-    def compute_possible_days(vrp)
-      # for each of this service's visits, computes first and last possible day to be assigned
-      vrp.services.each{ |service|
-        day = @schedule_start
-        nb_services_seen = 0
-
-        # first possible day
-        while day <= @schedule_end && nb_services_seen < service.visits_number
-          if service.unavailable_days.include?(day) || vrp.vehicles.none?{ |v| v.available_at(day) }
-            day += 1
+    def possible_days_from_possible_day(service, possible_days, vehicles, lapse, options)
+      day = @schedule_start
+      nb_services_seen = 0
```

Comment:
visit_index

Reply:
I would rather keep it nb_services_seen (or equivalent) because visit_index belongs to [1, nb_visits] while nb_services_seen belongs to [0, nb_visits]

Reply:
No, [0, nb_visits] :) It is just a function to compute the first/last possible day for each service's visit. We need to keep track of the number of visits seen within this function : we start with 0 (no visit has been assigned to a day) to nb_visits (all visits have been assigned to a day).